examples/00-Overview/02-get_data_from_static_simulation.py

```diff
@@ -0,0 +1,97 @@
+"""
+.. _ref_get_data_from_static_simulation:
+
+Get data from static simulation
+===============================
+This example show how to request data from a previously stored static simulation.
+The available results can be listed to see what results can be retrieved.
+"""
+
+###############################################################################
+# Imports and loading simulation
+# ------------------------------
+from ansys.dpf.post import examples, load_simulation
+
+simulation = load_simulation(examples.download_all_kinds_of_complexity())
+
+###############################################################################
+# Get and plot displacements
+# --------------------------
+disp = simulation.displacement()
+
+###############################################################################
+# Print information
+print(disp)
+
+###############################################################################
+# Plot displacements
+disp[0].plot()
+
+###############################################################################
+# Get and plot stresses
+# ---------------------
+# Request "XY" stress component
+stress = simulation.nodal_stress(component="XY")
+
+###############################################################################
+# Print information
+print(stress)
+
+###############################################################################
+# Plot available stresses
+stress[0].plot()
+stress[1].plot()
```

Comment:
Here, we could add some comments to remind the difference between the 2 stresses.